### PR TITLE
Add redshift subnet option

### PR DESCRIPTION
**Pull request by ChairmanTubeAmp**

### CODE REVIEW

**Comment by ChairmanTubeAmp** [comment]
Sorry about this mess of a PR. I should've made a branch after my first PR.

File: main.tf [hunk 1]
```diff
@@ -17,7 +17,7 @@ resource "aws_vpc_dhcp_options" "this" {
   count = "${var.enable_dhcp_options ? 1 : 0}"
 
   domain_name          = "${var.dhcp_options_domain_name}"
-  domain_name_servers  = "${var.dhcp_options_domain_name_servers}"
+  domain_name_servers  = [ "${var.dhcp_options_domain_name_servers}" ]
```
Comment (antonbabenko) [inline main.tf]: Why is this needed? The rest looks great.

Comment (ChairmanTubeAmp) [reply main.tf]: That fixed this: 
```$ terraform plan -out my.plan
1 error(s) occurred:

* module.vpc.aws_vpc_dhcp_options.this: domain_name_servers: should be a list

Comment (antonbabenko) [reply main.tf]: _I am back into business_ and reviewing outstanding PRs.

I still don't think this change is necessary because it is working without it and was working before. Please remove this and I will merge it right away.

Let us give it a try once more after this PR is merged by running `examples/complete-vpc`.

Comment (ChairmanTubeAmp) [reply main.tf]: It wasn't working without it for me.

**Comment by antonbabenko** [comment]
Great work! [v1.14.0](https://github.com/terraform-aws-modules/terraform-aws-security-group/releases/tag/v1.14.0) is now released via #54.
